installer/uninstaller.sh

```diff
@@ -44,6 +44,7 @@ K8S_NS_ARR=()
 
 cd ../scripts
 ./imm -k rm t default-tenant
+rm -rf ./certs/$IMM_RELEASE_PREFIX
```

Comment:
maybe we should ask for additional confirmation, for removing Certs? Maybe someone wants to create new platform with the same certs. Like in #58 `Add option to skip certificate generation, or document to reuse existing certs`